### PR TITLE
:memo: Add n as an option to manage Node versions

### LINKED ISSUE #2086: :memo: Add n for managing Node.js versions
### Where on docs.platform.sh should be changed?

https://docs.platform.sh/languages/nodejs/nvm.html

### What exactly should be updated?

Add example on using [`n`](https://github.com/tj/n) to manage Node.js versions, not just nvm.

Also make it clear that neither option should be used on Node.js images.

### Additional context

[Context](https://github.com/orgs/platformsh/projects/3)

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

Closes #2086

## What's changed

- Renamed file for Node.js versions to make more generic
- Updated links to file
- Added instructions for using n

### CODE REVIEW

File: docs/src/languages/nodejs/node-version.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        set -e
+
+        # Install the version specified in the .nvmrc file
+        n auto
+
+        # Reset the location hash to recognize the newly installed version
+        hash -r
+```
+
+## Use nvm
```
Comment (trolologuy) [inline docs/src/languages/nodejs/node-version.md:96]:
```suggestion
## Use `nvm`
```

File: docs/src/languages/nodejs/node-version.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+[Node Version Manager (nvm)](https://github.com/nvm-sh/nvm))
```
Comment (trolologuy) [inline docs/src/languages/nodejs/node-version.md:98]:
```suggestion
[Node Version Manager (`nvm`)](https://github.com/nvm-sh/nvm)
```

File: docs/src/languages/nodejs/node-version.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+You may have instances where you need to use a specific version of Node.js that isn't available in an application container.
+For example, a container might have a long-term support version,
+while you want the latest version.
+Or you want to use specific Node.js version inside a container for another language, such as PHP.
+
+In such cases, use a version manager to install the specific version you want to use.
+You could use one of these:
+
+* [`n`](https://github.com/tj/n)
+* [nvm](#use-nvm)
```
Comment (trolologuy) [inline docs/src/languages/nodejs/node-version.md:19]:
```suggestion
* [`nvm`](#use-nvm)
```

**Review by trolologuy — APPROVED** [review]
- [x] The [final built page](http://pr-2136-dw3q4sq-652soceglkw4u.eu-3.platformsh.site//configuration/services/headless-chrome.html) is working as expected.
- [x] The information is correct.
- [x] The information is comprehensive.
- [x] The information is concise.
- [x] The information is clear.
- [x] If anything has been moved, there are clear redirects.
- [x] All links work.
- [x] The changes follow the content and markup style guides.
- [x] Any relevant issues raised by the linting have been addressed.

**Comment by CollierCZ** [comment]
Hmm. I'd gone with the styling [under it's readme](https://github.com/nvm-sh/nvm#about), but it seems that first instance was a mistake on their part.

**Review by trolologuy — APPROVED** [review]

**Pull request merged**